Auth0/Authentication.swift

```diff
@@ -1080,6 +1080,57 @@ public protocol Authentication: Trackable, Loggable {
      */
     func jwks() -> Request<JWKS, AuthenticationError>
 
+    /**
+     Performs a custom token exchange to obtain Auth0 credentials using an existing identity provider token.
+
```

Comment:
Please add a notice about EA like https://github.com/auth0/Auth0.swift/blob/master/Auth0/Authentication.swift#L499-L503

Reply:
We should, yes.

Reply:
done